src/Connection.ts

```diff
@@ -72,7 +72,7 @@ export class Connection extends EventEmitter {
     return this._url;
   }
 
-  send<ReturnType extends {}>(method: string, params = {}): Promise<ReturnType> {
+  send<T extends keyof Protocol.CommandParameters>(method: T, params?: Protocol.CommandParameters[T]): Promise<Protocol.CommandReturnValues[T]> {
```

Comment:
this is an improvement on the types that I missed previously that was highlighted when migrating src/Browser

Reply:
neat!